org.neverball.Neverball.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: org.neverball.Neverball
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: neverball
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --persist=.neverball
+modules:
+  - name: physfs
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: http://icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
+        sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+  - name: neverball
+    buildsystem: simple
+    build-options:
+      ldflags: -z muldefs
+    sources:
+      - type: archive
+        url: https://github.com/Neverball/neverball/archive/neverball-1.6.0.tar.gz
+        sha256: 737377830aaa2069e00e2e5e4434c5578efda40d5dacb38d715643e52d9753bc
+      - type: patch
+        paths:
+          - makefile.patch
+          - version.patch
+      - type: file
+        path: org.neverball.Neverball.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I think this does not matter that much. Even GNOME Software itself still use appdata and when I use `/app/share/metainfo` directory, flatpak-builder moves the appdata file to `/app/share/appdata`.